src/main/java/com/extrahardmode/ExtraHardMode.java

```diff
@@ -79,14 +77,10 @@ public void onEnable()
         // Register modules
         registerModule(RootConfig.class, new RootConfig(this));
         registerModule(MessageConfig.class, new MessageConfig(this));
-
-        File rootFolder = new File(getDataFolder().getPath() + File.separator + "persistence" + File.separator);
```

Comment:
Um, could you at least maintain the same order more or less? Makes the diffs easier to follow.